tests/test_secondary.py

```diff
@@ -701,6 +724,74 @@ def test_40_process_metadata(self):
 
 
 
+  def test_45_process_partial_metadata(self):
+    """
+    Tests uptane.clients.secondary.Secondary::process_partial_metadata()
+
+    Tests PV Secondary client in 2 situations:
+     - secondary_instances[3]: Director's targets metadata available with 
+       valid signatures in JSON mode
+     - secondary_instances[3]: Director's targets metadata available with 
+       invalid signatures in JSON mode
+     - secondary_instances[3]: Director's targets metadata available with 
+       valid signatures in DER mode
```

Comment:
We should test the same scenarios in JSON and DER modes. The test suites run twice, once with `tuf.conf.METADATA_FORMAT` set to 'json' and once with `tuf.conf.METADATA_FORMAT` set to 'der'. As written, this code doesn't test an invalid signature in DER mode, ignores `tuf.conf.METADATA_FORMAT`, and runs the three tests twice the exact same way (when the full test suite is run by runtests.py in JSON mode, and once when the full test suite is run by runtests.py in DER mode).